exporters/otlp-metrics/build.gradle

```diff
@@ -0,0 +1,30 @@
+plugins {
+    id "java"
+    id "maven-publish"
+
+    id "ru.vyarus.animalsniffer"
+}
+
+description = 'OpenTelemetry Protocol Metrics Exporter'
+ext.moduleName = "io.opentelemetry.exporter.otlp.metrics"
+
+version = "${version}".replaceFirst(/^(\d+)\.(\d+).(\d+)/) { _, major, minor, patch ->
+    "${major}.${minor}.${patch}-alpha"
+}
```

Comment:
heh. now that we have this in 3 places, we should ponder how to not repeat this. No need to change it in this PR, but something to definitely think about.